docs/next/guides/accessories-and-menus/keyboard-shortcuts.md

```diff
@@ -280,4 +206,161 @@ hot.addHook('beforeKeyDown', (event) => {
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**X**</kbd> | <kbd>**Cmd**</kbd><br>+<br><kbd>**X**</kbd> | Cut the contents of the selected cells to the system clipboard  | &check; | &check; |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**C**</kbd> | <kbd>**Cmd**</kbd><br>+<br><kbd>**C**</kbd> | Copy the contents of the selected cells to the system clipboard | &check; | &check; |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**V**</kbd> | <kbd>**Cmd**</kbd><br>+<br><kbd>**V**</kbd> | Paste from the system clipboard                                 | &check; | &check; |
+
+#### Cell merging keyboard shortcuts
+
+These keyboard shortcuts work when the [`MergeCells`](@/api/mergecells.md) plugin is enabled.
+
+| Windows                                      | macOS                                        | Action                              |  Excel  | Sheets  |
+| -------------------------------------------- | -------------------------------------------- | ----------------------------------- | :-----: | :-----: |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**M**</kbd> | <kbd>**Ctrl**</kbd><br>+<br><kbd>**M**</kbd> | Merge or unmerge the selected cells | &cross; | &cross; |
+
+#### Undo and redo keyboard shortcuts
+
+These keyboard shortcuts work when the [`UndoRedo`](@/api/undoredo.md) plugin is enabled.
+
+| Windows                                                                   | macOS                                                                    | Action               |  Excel  | Sheets  |
+| ------------------------------------------------------------------------- | ------------------------------------------------------------------------ | -------------------- | :-----: | :-----: |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Z**</kbd>                              | <kbd>**Cmd**</kbd><br>+<br><kbd>**Z**</kbd>                              | Undo the last action | &check; | &check; |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Y**</kbd>                              | <kbd>**Cmd**</kbd><br>+<br><kbd>**Y**</kbd>                              | Redo the last action | &check; | &check; |
+| <kbd>**Ctrl**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**Z**</kbd> | <kbd>**Cmd**</kbd><br>+<br><kbd>**Shift**</kbd><br>+<br><kbd>**Z**</kbd> | Redo the last action | &check; | &check; |
+
+#### Context menu keyboard shortcuts
+
+These keyboard shortcuts work in context menus. To activate them, enable the [`ContextMenu`](@/api/contextmenu.md) plugin.
+
+| Windows                  | macOS                    | Action                                                |  Excel  | Sheets  |
+| ------------------------ | ------------------------ | ----------------------------------------------------- | :-----: | :-----: |
+| Arrow keys               | Arrow keys               | Move one available menu item up, down, left, or right | &check; | &check; |
+| <kbd>**Page Up**</kbd>   | <kbd>**Page Up**</kbd>   | Move to the first item of the context menu or submenu | &check; | &cross; |
```

Comment:
I think it is worth adding to the first **visible** element for clarification.

Reply:
@aninde - for clarification, you mean:

1. Move to the first item of the context menu or submenu -> Move to the first **visible** item of the context menu or submenu

2. Move to the last item of the context menu or submenu -> Move to the last **visible** item of the context menu or submenu

Is that correct? If so, I'm updating both the docs and the specification.